docs/integration_resources/software/nuke.md

```diff
@@ -6,17 +6,84 @@ Shift provides a `nshift` utility script to properly bind the Shift UI to Nuke.
 
 ```python
 import sys
-sys.path.append("<path_to_your_shift_folder>")
-sys.path.append("<path_to_your_shift_folder>/thirdparty/python/Lib/site-packages")
+sys.path.append("<path_to_the_shift_installation_folder>")
+sys.path.append("<path_to_the_shift_installation_folder>/shift/thirdparty/python/Lib/site-packages")
 
 from shift.tools import nshift
 nshift.show()
 ```
 
+## Shift installation in Nuke
+
+To install Shift and open the Shift UI with a menu entry within Nuke, it is required to set up Shift in the user init and menu Python files from Nuke. Shift can be added to the user init and menu files from Nuke's preference folder (*<home directory>/.nuke*) or to a custom init and menu files inside a custom folder structure. If the directory where these files are stored is a path read by Nuke, they will be automatically executed on startup.
+
+
+### Init
+
+In the init.py file is required to configure the Shift basics to work in Nuke batch and interface modes. For that purpose is only required to add the Shift installation path to the environment variables. 
+
+```python
+import sys
+sys.path.append("<path_to_the_shift_installation_folder>")
+sys.path.append("<path_to_the_shift_installation_folder>/shift/thirdparty/python/Lib/site-packages")
+```
+
+>[!NOTE]
+> If the paths are added in the system or before opening Nuke, then this configuration step will not be required.
+
+
+### Menu
+
+In the menu.py file it is required to configure the Shift menu and toolbar entries in the Nuke UI.
+
+
+```python
+import os
+import nuke
+
+try:
+    import shift
+    shiftPath = os.path.dirname(shift.__file__)  # Get the directory location of Shift
+    # Build the path to the shift_icon for the toolbar
+    iconPath = os.path.join(shiftPath, "icons", "shift_icon_window.png")
+    shiftCode = "from shift.tools import nshift;nshift.show()"
+    # Create the toolbar entry
+    menubar = nuke.menu("Nodes")
+    shiftM = menubar.addMenu("Shift", icon=iconPath)
+    shiftM.addCommand("Shift UI", shiftCode, '')
+    # Create a Menu entry
+    menubar = nuke.menu("Nuke")
+    shiftM = menubar.addMenu("Shift")
+    shiftM.addCommand("Shift UI", shiftCode, '')
+except Exception as e:
+    pass  # Skip the Error raise to avoid Nuke failing at opening if setting up the menu does not work on startup.
+```

```

Comment:
Update to use code block formatting 
https://github.com/Inbibo/shift_docs/issues/60#issuecomment-2326086193